docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md

```diff
@@ -0,0 +1,91 @@
+
+## Using Inspect View Tree for Real Devices
+
+By default, the Inspect View Tree feature is enabled if the instrumentation feature is enabled.
+
+To enable device instrumentation:
+
+1. On Sauce Labs, click **Live** > **Mobile App**.
+2. [Upload your mobile app to Sauce Labs](/mobile-apps/app-storage/) through the UI or CLI.
+3. After you’ve uploaded your app, return to the **Live** > **Mobile App** page, hover your mouse over your app, then select **Settings**. <br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr.png')} alt="Mobile app settings navigation" width="600"/>
+4. Under **Default Settings**, toggle **Instrumentation** on.
+   <br/><img src={useBaseUrl('img/mobile-apps/Device-Vitals-screenshot-1.png')} alt="Mobile app settings navigation" width="600"/>
+
+Now you can start your live or automated testing session. View Tree will be captured and be part of your test results.
+
+:::note
+The Inspect View Tree feature works for all the automated test framework types.
+:::
+
+## Accessing Inspect View Tree
+
+You can access the Inspect View Tree from the [Live Test Results](/mobile-apps/live-testing/live-mobile-app-testing/) or [Automated Test Results](/mobile-apps/automated-testing/) page.
+
+1. In Sauce Labs, click **LIVE** or **AUTOMATED** and then click **Test Results**.  
+<br/><img src={useBaseUrl('img/mobile-apps/DEvice-Vitals-screenshot-2.png')} alt="Mobile app settings navigation" width="200"/>
+<br/><img src={useBaseUrl('img/mobile-apps/DEvice-Vitals-screenshot-7.png')} alt="Mobile app settings navigation" width="200"/>
+
+2. Select a test result from the list.
+<br/><img src={useBaseUrl('img/mobile-apps/view-tree-screenshot-1.png')} alt="Mobile app settings navigation" width="800"/>
+
+3. Click on **Inspect** button.
+<br/><img src={useBaseUrl('img/mobile-apps/view-tree-screenshot-2.png')} alt="Mobile app settings navigation" width="600"/>
+
+The Inspect View Tree feature will display the captured snapshots of detectable view changes:
+
+<br/><img src={useBaseUrl('img/mobile-apps/view-tree-screenshot-3.png')} alt="Mobile app settings navigation" width="700"/>
+
+
+You can also download the results using the API call below:
+
+ ```java
+  curl --compressed \
+-O https://{SAUCE_USERNAME}:{SAUCE_ACCESS_KEY}@{DATA_CENTER}.saucelabs.com/v1/rdc/jobs/{JOB_ID}/insights.json
+  ```
+
+### Inspect View Tree Attributes
+
+The following table lists the available attributes:
+
+|Name|Description|
+|:---|:---|
+|ID|The class identifier|
+|Visibility| A value that checks if the component is present in the View Tree|
+|Focusable|A value that provides additional information associated with accessibility|
+|Alpha|A value that describes how translucent or opaque the component is|
+|Enabled|Checks the interactivity of the component|
+|ContentDesc|The accessibility description of an UI component|
+|Text|A text attribute of an UI component|
+
+## Limitations
+
+:::note
+* Webviews and hybrid apps will have additional capture limitations.
```

Comment:
What are these limitations?

Reply:
@lysannep I spoke with @kristofmuhi   about mentioned limitations and the team does not know which exact details are missing from the capture of webviews, they know that some buttons and views might be missing, but this information is hidden. So the selected sentence under "limitations" does not include a detailed explanation.